test/e2e/virtual/apiexport/authorizer_test.go

```diff
@@ -453,8 +459,8 @@ metadata:
 		if err == nil {
 			return false, "expected error, got none"
 		}
-		if !apierrors.IsNotFound(err) {
-			return false, fmt.Sprintf("expected a not-found error, but got %v", err)
+		if !apierrors.IsForbidden(err) {
```

Comment:
@s-urbaniak @sttts not sure if this is right, but when it fails in CI, it gets forbidden for wait.ForeverTestTimeout. So I'm guessing the NotFound happens initially (or never) and then transitions to forbidden (eventual consistency). Would like to understand the "why" though.

Reply:
agreed, this seems to be a temporary state before final replication of logical clusters finalizes and needs investigation.

Reply:
correction: if the "Forbidden" is a temporary state than it should converge against "IsNotFound" which is not the case.